cmd/integrations.go

```diff
@@ -0,0 +1,136 @@
+package cmd
+
+import (
+	"github.com/urfave/cli"
+
+	"github.com/Scalingo/cli/cmd/autocomplete"
+	"github.com/Scalingo/cli/integrations"
+)
+
+var (
+	IntegrationsListCommand = cli.Command{
+		Name:     "integrations",
+		Category: "Integrations",
+		Usage:    "List your external integrations",
```

Comment:
Not sure "integrations" is the best word for the customer to understand what it's doing. Maybe "SCM integrations"?